### ISSUE TITLE
`import/no-unresolved` ignores require statements with backticks

### ISSUE THREAD

**Issue by taylorbryant** [import-js/eslint-plugin-import#1575]
First of all, thanks for this plugin! It's great!

We're using the `backtick` option with the [`quotes` rule](https://eslint.org/docs/rules/quotes#options) in our codebase. About a week ago, we realized that our linter wasn't catching bad imports. After scratching our heads for a bit, we realized that `import/no-unresolved` doesn't check require statements that use backticks.

```javascript
// index.js
// Assume a, b, and c do not exist

// Does not result in a linting error
const sweetPotato = require(`a`);

// Shows linting error
const biscuit = require("b");

// Shows linting error
import babyYoda from "c";
```
